### PR TITLE
Release 1.1

### PR DESCRIPTION
**Pull request by maorlipchuk**

### CODE REVIEW

**Comment by maorlipchuk** [comment]
ci please build

**Comment by maorlipchuk** [comment]
ci please build

**Comment by maorlipchuk** [comment]
ci please build

**Comment by mwperina** [comment]
Maor, are you sure that you want to switch the release from alpha (version 0.y) to stable (1.y)? Shouldn't the https://github.com/oVirt/ovirt-ansible-disaster-recovery/issues/17 be either resolved or closed before releasing stable version?

@maorlipchuk

File: build.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-VERSION="0.2"
-MILESTONE=master
-RPM_RELEASE="0.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
+VERSION="1.1"
```
Comment (machacekondra) [inline build.sh]: Please use `1.1.0`

File: build.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+MILESTONE=
+RPM_RELEASE="1.1$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
```
Comment (machacekondra) [inline build.sh]: RPM_RELEASE="1"

Comment (machacekondra) [reply build.sh]: in post release patch you should have:

RPM_RELEASE="0.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"

**Pull request closed without merging**